src/transaction.rs

```diff
@@ -124,6 +124,7 @@ where
 
         let new_tx = new_transaction(self.deps.driver.clone());
         self.current_transaction.replace(new_tx);
+        self.timer.start();
```

Comment:
I don't think we should start the timer here. The current transaction will actually start at https://github.com/awslabs/amazon-qldb-shell/blob/df08c285286b403afc4a9ae935d00805904a1309/src/transaction.rs#L55, so we should start the timer right after the current transaction starts, which will be
```
let outcome = driver
            .transact(|mut tx| async {
                Timer::new().start();
                loop {
                ...
```